sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -354,7 +354,16 @@ export class TextAnalyticsClient {
         code: CanonicalCode.UNKNOWN,
         message: e.message
       });
-      throw e;
+      if (e.response.parsedBody.error.innererror.code === "InvalidDocumentBatch") {
```

Comment:
This looks suspiciously effortless to reach this far into the object. Is something in this chain `any` (maybe `parsedBody`)? If so, we should do more validation here to make sure the structure is what we expect.

Reply:
done.